internal/ifelse/rule.go

```diff
@@ -31,8 +31,11 @@ type Rule interface {
 func Apply(rule Rule, node ast.Node, target Target, args lint.Arguments) []lint.Failure {
 	v := &visitor{rule: rule, target: target}
 	for _, arg := range args {
-		if arg == PreserveScope {
+		switch arg {
+		case PreserveScope:
 			v.args.PreserveScope = true
+		case AllowJump:
+			v.args.AllowJump = true
 		}
 	}
```

Comment:
Isn't simply this? 🤔

```go
         v.args.PreserveScope = slices.Contains(args,  PreserveScope)
         v.args.AllowJump = slices.Contains(args,  AllowJump)
```

Reply:
Right, with one less loop over the slice